.github/workflows/ci.ym

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
I would expect this to trigger the CI for any PR that targets `main`, but it looks to me like this PR didn't trigger the CI, and that it kept not working until the changes in #6.

Reply:
Yes, at first I had named my yaml file ".ym" instead of ".yml" and the tutorial i followed didn't behave the way I intended it to, so I made many small commits which makes it hard to follow the development flow.

If you have any tips on how to test something like this without having to actually push and merging it first I would appreciate to hear about it.

Reply:
Wow I didn't even realize the file name was incorrect. Thanks for pointing that out.

I'm afraid there is no other way to reliably test GitHub actions than to push to a PR, but I think that's fine. Any work you do in that PR will have immediate effect on the actions for that PR, and once you're happy you can squash and merge so that it looks in the Git history as if you only made a single commit.